install/pihole-install.sh

```diff
@@ -73,7 +74,6 @@ server:
   infra-cache-slabs: 8
   key-cache-slabs: 8
   serve-expired: yes
-  root-hints: /var/lib/unbound/root.hints
```

Comment:
root hints should still be added for recursive configuration

Reply:
Read description

Reply:
According to the Unbound documentation if root.hints is supplied by the dns-root-data package (as installed by Pi-hole) there is no need to declare it via root-hints.  As noted /var/lib/unbound/root.hints is grabbed once at install but never updated.  Using the one provided by dns-root-data (/usr/share/dns/root.hints) the package manager will update it as needed.

Reply:
We could still do an explicit declaration if you prefer?

root-hints: /usr/share/dns/root.hints